openshift/README.md

```diff
@@ -31,7 +31,10 @@ wget $RESOURCES_URL -qO - \
 https://radanalytics.io/resources.yaml  
 This is the latest resources file from [radanalytics.io](https://radanalytics.io)  
 DEFAULT_TAG=stable  
-NEW_TAG may be any version newer than v0.4.0  
+NEW_TAG may be any version newer than v0.5.3
+
+https://radanalytics.io/openshift/resources-v0.5.3.yaml  
+This file supports version v0.5.3 of Oshinko  
```

Comment:
Not sure of the best way to phrase this consistent with the other entries, but this should communicate that you can use this template with any version from 0.4.1 to 0.5.3 inclusive

Reply:
Okay, I think it should read something like this to match what we did for v0.3.yaml

This file supports versions v0.4.x through v0.5.x of Oshinko
DEFAULT_TAG=v0.5.3
NEW_TAG may be any version from v0.4.1 to v0.5.3 inclusive